packages/roosterjs-editor-core/lib/coreAPI/insertNode.ts

```diff
@@ -31,18 +34,16 @@ function getInitialRange(core: EditorCore, option: InsertOption): { range: Range
         rangeToRestore = range.cloneRange();
     }
 
-    return {range, rangeToRestore };
+    return { range, rangeToRestore };
 }
 
 const insertNode: InsertNode = (core: EditorCore, node: Node, option: InsertOption) => {
-    if (!option) {
-        option = {
-            position: ContentPosition.SelectionStart,
-            insertOnNewLine: option.insertOnNewLine != null ? option.insertOnNewLine : false,
-            updateCursor: option.updateCursor != null ? option.updateCursor : true,
-            replaceSelection: option.replaceSelection != null ? option.replaceSelection : true,
-        }
-    }
+    option = option || {

```

Comment:
@Adjective-Object mentioned in the last PR that this would not work, because if option is non-falsey but has nulled/undefined parameters, we wouldn't be providing the correct set of defaults. What JS error are we guarding against, and is there a way we can guarantee the intent of having a default set of a parameters?

Reply:
old code is "if (!option) { option.***)“

here option is null. it repros 100% when auto bullet

Reply:
If user passes option.something = null, we will treat it as false. This is by design.